packages/components/form/__specs__/input.spec.tsx

```diff
@@ -0,0 +1,99 @@
+import { render, waitFor } from "../test_utils";
+import { Search } from "iconoir-react";
+import { theme } from "@buoysoftware/anchor-theme";
+
+import { Input } from "../src";
+
+describe("<Input />", () => {
+  it("renders properly", () => {
+    const { asFragment } = render(<Input id="example" name="example" />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("renders a disabled input properly", () => {
+    const { asFragment } = render(
+      <Input id="example" disabled={true} name="example" />
+    );
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  context("unit", () => {
+    it("renders a unit properly", () => {
+      const { asFragment } = render(
+        <Input id="example" unit="mL" name="example" />
+      );
+
+      expect(asFragment()).toMatchSnapshot();
+    });
+
+    it("does not apply any spacing", () => {
+      const { getByTestId } = render(
+        <Input id="example" unit="mL" name="example" />
+      );
+
+      expect(getByTestId("input-unit")).toHaveStyleRule("margin-right", "0rem");
+    });
+  });
+
+  context("prefix unit", () => {
+    it("renders a prefix unit properly", () => {
+      const { asFragment } = render(
+        <Input id="example" unit="$" prefixUnit={true} name="example" />
+      );
+
+      expect(asFragment()).toMatchSnapshot();
+    });
+
+    it("spaces the prefix unit properly", () => {
+      const { getByTestId } = render(
+        <Input id="example" unit="$" prefixUnit={true} name="example" />
+      );
+
+      expect(getByTestId("input-unit")).toHaveStyleRule(
+        "margin-right",
+        theme.space.xs
+      );
+    });
+  });
```

Comment:
Is the prefix something we are intentionally carrying over as part of the new design system? I believe the only place they are currently used is the facility config which doesn't follow any of our designs.

Reply:
Erik has it built into the new design system so it is intentional.

Reply:
Sweet!